cmd/statusd/main.go

```diff
@@ -318,6 +190,24 @@ func configureStatusService(flagValue string, nodeConfig *params.NodeConfig) (*p
 	return nodeConfig, nil
 }
 
+func parseConfig(configFiles configFlags, config *params.NodeConfig) bool {
+	configSpecified := len(configFiles) > 0
+	if configSpecified {
+		// Merge specified configuration files, in order
+		if err := params.ReadConfigFiles(configFiles, config); err != nil {
```

Comment:
Not sure I understand your suggestion. Do you mean we should not check the length of `configFiles` and instead just call `ReadConfigFiles`? I decided not to allow running the app without passing at least one config file, since the alternative of having everything specified with env vars doesn't look really useful.

Reply:
>  Do you mean we should not check the length of configFiles and instead just call ReadConfigFiles?

Correct. But we do have some default values when calling `NewNodeConfig`. Is this not enough to start the node?

Reply:
@adambabik does this mean that we'll still want to populate the `ClusterConfig.BootNodes` with the values for `eth.beta` as in current `develop`?

Reply:
Hm I would like to say no. To start it with `eth.beta`, one would need to do `./statusd -c beta-fleet.json`. However, now the question is what's the point of running `statusd` without any peers? 

So maybe you're right that it should not be possible to start `statusd` without a config but we should provide some predefined JSON configs in the repo.